assets/scss/footer.scss

```diff
@@ -1,20 +1,41 @@
 footer {
+  background: $midnight;
+  color: $white;
+
+  h5 {
+    font-size: $font-size-small;
+    color: $veryLightGrey;
```

Comment:
The greys in the variables.scss are the greys from the styleguide. I want to try to stick to those, to avoid having too much shades of (one time used) greys. Would you mind using one of the brand greys or just white?